### ISSUE TITLE
'tcpflow -b 0' now works

### ISSUE THREAD

**Issue by dkogan** [simsong/tcpflow#95]
Hi. Here's a patch for https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=638804

http://pub.secretsauce.net/0001-tcpflow-b-0-now-works.patch

**Comment by simsong** [comment]
Please submit this as a pull request.

**Comment by dkogan** [comment]
Hi. Github is confused, and isn't allowing pull requests, but vanilla git works, as always. You can download that file, and 'git am 0001-tcpflow-b-0-now-works.patch' to import the patch into your tree.

**Comment by simsong** [comment]
I'd rather fix the pull request problem. What's the problem?

**Comment by md5** [comment]
@dkogan I don't see a fork in the members list for you: https://github.com/simsong/tcpflow/network/members

Do you have a fork of `tcpflow` under an organization?

**Comment by dkogan** [comment]
Apparently I do not. Look, let's use git. I gave you a patch earlier. If you don't want a patch, here's a github tree: https://github.com/dkogan/tcpflow/tree/fixed_-b0_works

I have no interest in fixing github, but this is more than enough to fix this small issue in tcpflow.

Thanks

**Comment by simsong** [comment]
I don't understand @md5's comment.  I've been very busy, but I can try to apply your git patch.

**Comment by simsong** [comment]
@dkogan, your patch failed with this error:

error: patch failed: src/tcpdemux.h:90
error: src/tcpdemux.h: patch does not apply
Patch failed at 0001 'tcpflow -b 0' now works
The copy of the patch that failed is found in:
   /Users/simsong/gits/tcpflow/.git/rebase-apply/patch
When you have resolved this problem, run "git am --continue".
If you prefer to skip this patch, run "git am --skip" instead.
To restore the original branch and stop patching, run "git am --abort".

I will manually apply your patch.

**Comment by md5** [comment]
@simsong At the time I made that comment, I don't think the repository fork at https://github.com/dkogan/tcpflow existed.

**Comment by dkogan** [comment]
Hi. Sorry for any confusion. Some notes:

1. The patch came from a git tree, so it did apply to the HEAD at the time it was generated. Since then, the HEAD changed a bit, and you're seeing a conflict. The git tree I posted today is already rebased, so it should be "right".

2. The repository I posted today is indeed new. The old one had an issue where github didn't recognize it as a fork of tcpflow, so it made pull requests not work. Since then, that old repository was blown away, and a new one made (both contain the same commits). The new one has the same issue, however.
